modules/microshift-lvmd-yaml-creating.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * microshift_storage/microshift-storage-plugin-overview.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-lvmd-yaml-creating_{context}"]
+= Creating an LVMS configuration file
+
+When {product-title} runs, it uses LVMS configuration from `/etc/microshift/lvmd.yaml`, if provided. You must place any configuration files you make into the `etc/microshift/` directory.
+
+.Procedure
+
+* To create the `lvmd.yaml` configuration file, run the following command:
+
+[source, terminal]
+----
+$ sudo cp /etc/microshift/lvmd.yaml.default /etc/microshift/lvmd.yaml
+----
+<2> Volume group. You can set a custom volume group name here.
```

Comment:
This looks like a reference to something in the old example. I think we can delete the line?

```suggestion
```